sqe-http-api/Services/UserService.cs

```diff
@@ -104,16 +122,110 @@ public UserDTO GetCurrentUser()
             return null;
         }
 
+        /// <summary>
+        /// This will create a new user account in the database and email an authorization token to the user.
+        /// </summary>
+        /// <param name="newUserData">All of the information for the new user.</param>
+        /// <returns>Returns a UserDTO for the newly created user account.</returns>
+        public async Task<UserDTO> CreateNewUserAsync(NewUserDTO newUserData)
+        {
+            // Ask the repo to create the new user
+            var createdUser = await _userRepository.CreateNewUserAsync(newUserData.userName, newUserData.email, newUserData.password,
+                newUserData.forename, newUserData.surname, newUserData.organization);
+            
+            // Email the user
+            // TODO: Add link to web endpoint when we know what that is. Can token be in URL query?
+            const string emailBody = @"
+<html><body>Dear $User,<br>
+<br>
+Thank you for registering with the Scripta Qumranica Electronica research platform.  The token
+to activate your new account is: $Token.<br>
```

Comment:
Also, don't place tokens in the email, you should put a link that user clicks (I understand you used this for debugging purposes)

Reply:
Agreed (see the TODO in line 137).